src/docs/redis/sections/10-try/20-server.mdx

```diff
@@ -0,0 +1,25 @@
+You can test your certificate by starting up Redis with TLS enabled.
+
+
+<Alert severity="warning">
+  Native "SSL Support" (TLS) was added to Redis 6.0.0, which was released GA on April 30, 2020.
+  TLS in Redis is an <strong>optional feature</strong> and not all Redis server binaries will have it enabled.
+  The official Docker image, and many OS distribution images, have TLS enabled.
```

Comment:
I haven't looked at this rendered yet, but sometimes `Alert` content needs a wrapping `<div>` if it contains nested jsx/html components for styles to apply properly. Disregard if it looks fine.